Logger/LogFactory.cs

```diff
@@ -1,10 +1,20 @@
-﻿namespace Logger;
-
-public class LogFactory
+﻿namespace Logger
 {
-    public BaseLogger CreateLogger(string className)
+    public class LogFactory
     {
-        
-        return null;
+        private string? FilePath { get; set; }
```

Comment:
```suggestion
        private string? _FilePath;
```
This doesn't need to be an auto property, it can just be a private field.